src/app/info/InfoPage.tsx

```diff
@@ -0,0 +1,69 @@
+"use client";
+
+import React, { useState, useEffect } from "react";
+import WikiAccordian from "./WikiAccordian";
+
+import { Supabase } from "@/supabaseUtils";
+import { DbWikiRow } from "@/databaseUtils";
+import supabase from "@/supabaseClient";
+
+const getTopWikiRow = async (supabase: Supabase): Promise<DbWikiRow> => {
+    const query = (await supabase.from("wiki").select("*").limit(1).single()) as {
+        data: DbWikiRow;
+        error: Error | null;
+    };
+
+    if (query.error) {
+        throw query.error;
+    }
+    return query.data;
+};
+
+export const formatContent: (topRowContent: string) => (string | React.JSX.Element)[] = (
+    topRowContent: string
+) => {
+    const topRowContentParts = topRowContent.split(/(\[.*\]\(.*\)|<.*>)/);
+    let counter = 0;
+    const formattedContent = topRowContentParts.map((part, index) => {
+        if (index % 2 === 1) {
+            const plainPart = part.slice(1, -1);
+            if (plainPart.includes("](")) {
+                const items = plainPart.split(/\]\(/);
+                return (
+                    <a href={items[1]} key={counter++}>
+                        {items[0]}
+                    </a>
+                );
+            } else {
+                return (
+                    <a href={plainPart} key={counter++}>
+                        {plainPart}
+                    </a>
+                );
+            }
+        } else {
+            return part;
+        }
+    });
+    return formattedContent;
+};
+
+const InfoPage: React.FC = () => {
+    const [topRow, setTopRow]: [DbWikiRow, (a: DbWikiRow) => void] = useState({
+        title: "",
+        content: "",
+        order: -1,
+    });
```

Comment:
Can we initialise this to `null`? It forces better type safety and makes comparisons more clear. Also, the correct way to type state is:
`const [state, setState] = useState<Type>(initial_value)`